src/components/_tests_/cameraButton.spec.ts

```diff
@@ -0,0 +1,64 @@
+import { shallow } from "enzyme";
+import { createElement } from "react";
+import { CameraButton, CameraButtonProps } from "../CameraButton";
+
+describe("CameraButton", () => {
+    const createCameraButton = (props: CameraButtonProps) => shallow(createElement(CameraButton, props));
+    const defaultProps: CameraButtonProps = {
+        buttonLabel: "Take photo",
+        caption: "buttons",
+        glyphIcon: "camera",
+        onClickAction: jasmine.createSpy("onClick"),
+        spanClass: "widget-camera-picture"
+    };
+
+    describe("with action labels as 'buttons'", () => {
+        it("should render the structure correctly", () => {
+            const cameraButton = createCameraButton(defaultProps);
+            expect(cameraButton).toBeElement(
```

Comment:
Test should separated into 2 or 3 parts
-Initialization / Declaration
-Execution
-Tests

So add space

Reply:
Done